bin/asciidoctorjs

```diff
@@ -19,10 +19,14 @@ var convert_options = function (cliOptions) {
   var sectionNumbers = cliOptions['section-numbers'];
   var baseDir = cliOptions['base-dir'];
   var destinationDir = cliOptions['destination-dir'];
+  var requireLib = cliOptions['require'];
   var quiet = cliOptions['quiet'];
   var verbose = cliOptions['verbose'];
   var timings = cliOptions['timings'];
   var trace = cliOptions['trace'];
+  if (requireLib) {
+    require(requireLib);
```

Comment:
@mojavelinux What do you think about this basic implementation ? Is it too simple ? Should we use what Antora is using: https://gitlab.com/antora/antora/blob/4f84db7edca0f859ee443d2f2dd74887ee97975c/packages/cli/lib/cli.js#L26-40 ?

Reply:
I think it's a good place to start. I plan to extra the logic in Antora you referenced into a reusable library. We should probably wait until that's available to introduce more advanced logic.